contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/test-plan-ce-oem.pxu

```diff
@@ -82,6 +83,7 @@ nested_part:
     com.canonical.certification::rtc-automated
     com.canonical.certification::led-indicator-auto
     before-suspend-ce-oem-spi-automated
+    ce-oem-secure-boot-automated
```

Comment:
It should be better to add some description for this empty test plan

Reply:
I will not include `ce-oem-secure-boot-automated` test plan since it's empty. Thanks